index.html

```diff
@@ -4,11 +4,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Document </title>
     <link rel="stylesheet" href="style.css"> 
 </head>
 <body>
-    <h1>Hello  Microverse </h1>
+    <h1>Hello Microverse </h1>
 
     
```

Comment:
## 4.
### Your MIT.md file
![MIT](https://user-images.githubusercontent.com/93039359/184926415-8afd8e50-667b-4fb5-a65e-b03e355afd4f.png)

- [ ] Please add your name, current year, app type in your MIT.md file. :blush: because this is your license file please take it seriously :blush: